recipes/msgpack-numpy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "msgpack-numpy" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "afc603c7cf8497fb125a8c8c713518a004e9662101f088e3d4fcf7688b08eeb3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
It looks like this can be a `noarch: python` package:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages